docker-compose.override.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.4'
+
+services:
+  watch-gulp:
+    build:
+      context: .
+      target: gulp
+    command: bash -c "npx gulp assemble && npx gulp watch"
```

Comment:
Would like to find a way to remove the `assemble`, but due to the volume don't think it can be.

Reply:
A named volume would be initialized with the files from the container image, but since this is a host folder, this is needed.
A known limitation is that anything written by these tasks will be created with `root` ownership on certain platforms:
```
[15:44:50][giorgio@Newton:~/code/patterns-core]$ ls source/css/sass/variables/ -l
total 16
-rw-r--r-- 1 root root 239 Nov 30 15:41 _baselinegrid.scss
-rw-r--r-- 1 root root 149 Nov 30 15:41 _breakpoint.scss
-rw-r--r-- 1 root root 857 Nov 30 15:41 _color.scss
-rw-r--r-- 1 root root 361 Nov 30 15:41 _font.scss
[15:44:53][giorgio@Newton:~/code/patterns-core]$ ls source/css/sass/ -l
total 72
-rw-rw-r-- 1 giorgio giorgio    57 Nov 30 15:44 base.scss
-rw-rw-r-- 1 giorgio giorgio   295 Sep 18 14:42 build.scss
-rw-rw-r-- 1 giorgio giorgio    39 Sep 18 14:42 _definitions.scss
-rw-rw-r-- 1 giorgio giorgio   702 Sep 18 14:42 _fonts.scss
-rw-rw-r-- 1 giorgio giorgio  5011 Nov 30 15:33 _mixins.scss
-rw-rw-r-- 1 giorgio giorgio 11658 Nov 30 15:33 _mixins--spacing.scss
-rw-rw-r-- 1 giorgio giorgio  1665 Nov 30 15:33 _mixins--typography.scss
-rw-rw-r-- 1 giorgio giorgio  5114 Nov 30 15:33 _normalize-overrides.scss
-rw-rw-r-- 1 giorgio giorgio  8000 Sep 18 14:42 _normalize.scss
-rw-rw-r-- 1 giorgio giorgio    52 Sep 18 14:42 reset.scss
-rw-rw-r-- 1 giorgio giorgio  1716 Nov 30 15:33 _utility-functions.scss
drwxr-xr-x 2 root    root     4096 Nov 30 15:41 variables
-rw-rw-r-- 1 giorgio giorgio   172 Nov 30 15:33 _variables.scss
```
Maybe on Macs it's better because there's the additional layer of the Linux VM.